CHANGELOG.unreleased.md

```diff
@@ -17,7 +17,8 @@
 
 - [Rolling Pool Update] After the update, when migrating VMs back to their host, do not migrate VMs that are already on the right host [Forum#7802](https://xcp-ng.org/forum/topic/7802) (PR [#7071](https://github.com/vatesfr/xen-orchestra/pull/7071))
 - [RPU] Fix "XenServer credentials not found" when running a Rolling Pool Update on a XenServer pool (PR [#7089](https://github.com/vatesfr/xen-orchestra/pull/7089))
-- [Usage report] Fix "Converting circular structure to JSON" error
+- [Usage report] Fix "Converting circular structure to JSON" error (PR [#7096](https://github.com/vatesfr/xen-orchestra/pull/7096))
```

Comment:
I see you sneaked up an unrelated change in this PR :smirk: 

I did not see anything :speak_no_evil:

Reply:
It's related... to the usage-report plugin :innocent: